packages/stk/stk_unit_tests/stk_ngp_test/utest_VirtualFunction.cpp

```diff
@@ -60,13 +60,13 @@ class NgpDerived : public NgpBase
 
 struct SimpleStruct {
   KOKKOS_FUNCTION
-  void print() { printf("Printing from A located at %p\n", static_cast<void*>(this)); }
+  void print() { Kokkos::printf("Printing from A located at %p\n", static_cast<void*>(this)); }
 };
 
 struct BaseStruct {
   virtual void set_i(const int) = 0;
   KOKKOS_FUNCTION
-  virtual void print() { printf("Printing from base located at %p\n", static_cast<void*>(this)); }
+  virtual void print() { Kokkos::printf("Printing from base located at %p\n", static_cast<void*>(this)); }
 };
 
 struct ChildStruct : public BaseStruct {
```

Comment:
There's another `printf` that needs to be `Kokkos::printf` in this struct.

Reply:
Fixed.